index.html

```diff
@@ -13,7 +13,7 @@
 <body>
```

Comment:
- [ ] [Optional] Please consider increasing the font size of the nav links. Currently, it's not easily readable.

Reply:
@jasonfelice 
Thank you for the feedback! I have already updated the font size of the nav links to address this issue.